library/Freckle/App/Kafka/Producer.hs

```diff
@@ -91,13 +91,13 @@ createKafkaProducerPool
   -> KafkaProducerPoolConfig
   -> IO (Pool KafkaProducer)
 createKafkaProducerPool addresses KafkaProducerPoolConfig {..} =
-  Pool.newPool $
-    Pool.setNumStripes (Just kafkaProducerPoolConfigStripes) $
-      Pool.defaultPoolConfig
-        mkProducer
-        closeProducer
-        (realToFrac kafkaProducerPoolConfigIdleTimeout)
-        kafkaProducerPoolConfigSize
+  Pool.newPool
+    $ Pool.setNumStripes (Just kafkaProducerPoolConfigStripes)
+    $ Pool.defaultPoolConfig
+      mkProducer
+      closeProducer
+      (realToFrac kafkaProducerPoolConfigIdleTimeout)
+      kafkaProducerPoolConfigSize
```

Comment:
Don't worry about changing this right now, but I reread a little bit on [the docs here](https://hackage.haskell.org/package/resource-pool-0.4.0.0/docs/Data-Pool.html#v:defaultPoolConfig) and I'm not sure we ever want to `setNumStripes`. The default is to match the current capabilities, which is useful to ensure threads don't fight over the same stripe when pulling a value out of the pool. I can't imagine not wanting this. I think it'd be better to pretend stripes don't exist in our own pool configurations types.

I think we got confused because the docs talk about how size / stripes  might result in configured size `N` being actually-sized `N - 1` due to integer division, so maybe it's something someone might care about, but I really don't think we ever should.